REST-API-ADVANCED/advanced/src/main/java/com/epam/esm/security/dto/OidcDTO.java

```diff
@@ -0,0 +1,13 @@
+package com.epam.esm.security.dto;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Builder
+public class OidcDTO {
```

Comment:
What OIDC is standing for?

Reply:
Open id connection

Reply:
Do you think it makes sense to rename classes to have a full name?

Reply:
Sure. Renamed to OpenIdConnectionModel